CMakeLists.txt

```diff
@@ -123,6 +124,7 @@ else()
     find_package(Boost COMPONENTS system)
     target_link_libraries(boost_json
         PUBLIC
+            Boost::container
             Boost::system
```

Comment:
Why do we link to system here but not on line 96

Reply:
it's linked on line 100